### ISSUE TITLE
Playlist issue in headset

### ISSUE THREAD

**Issue by xAgent42x** [emulamer/BeatOn#6]
Creating a second playlist in headset does not allow custom image as Quest does not have a built in browser to do so.  Moving songs from one playlist to another causes playlist to lose songs.  Specifically play list one has two songs and playlist to has none, moving a song from PL1 to any other PL will make additional songs to disappear from PL1.  Only way to get songs back is to use Reload Songs Folder button and they reappear in Custom Songs PL.  This also happens on PC browser.  Also with 195 songs the selected song disappears from Custom Songs completely and a different song was put into the additional playlist.

**Comment by emulamer** [comment]
Most of this should be corrected now, although there's still some issues with loading tons of songs and dragging them all over the place.  Still doesnt support cover art on the quest, I'm not sure where you'd get the images from.

**Comment by xAgent42x** [comment]
Songs transfer but a song is still removed from the initial playlist that was not chosen to be moved. Moving songs after clicking sync button after creating new custom song playlist prevents songs from disappearing.  Need to ensure the syncing occurs before dragging songs.  However, songs were not added to custom playlist in game.  Doing it the same way a second time made the songs appear in game.

**Comment by xAgent42x** [comment]
After so many songs in the playlist they lose their cover album assets and trying to add songs with no photo do not trigger sync button and therefore do not sync to game.  I had 195 songs from dragging into folder and those loaded fine but once I started scrolling further down in beaton they started to not show assets.  I then went to the store to add another song and it had no assets.  Tried to add songs with no pictures to 3rd playlist.  Sync was not triggered and songs did not show in game.  Also additional song added from BeastSaber did not load into game at all in any of the 3 playlists.
